src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <>
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  </>
```

Comment:
Remove redundant react.fragment and destructure `user`
```suggestion
    <a className="UserInfo" href={`mailto:${user.email}`}>
      {user.name}
    </a>
```